Trading/Mode/daily_trading_mode/daily_trading_mode.py

```diff
@@ -197,9 +197,7 @@ def __get_market_quantity_from_risk(self, eval_note, quantity, quote, selling=Fa
         factor = self.QUANTITY_MARKET_MIN_PERCENT + (
                 (abs(eval_note) + weighted_risk) * self.QUANTITY_MARKET_ATTENUATION)
 
-        checked_factor = AbstractTradingModeConsumer.check_factor(self.QUANTITY_MARKET_MIN_PERCENT,
```

Comment:
I think there is multiple `AbstractTradingModeConsumer.check_factor` in this file isn't it ?